onecgiar-pr-client/src/app/pages/ipsr/pages/innovation-package-detail/pages/ipsr-innovation-use-pathway/pages/step-n2/pages/complementary-innovation/components/table-innovation/table-innovation.component.spec.ts

```diff
@@ -109,10 +118,11 @@ describe('TableInnovationComponent', () => {
     const alertFeSpy = jest.spyOn(mockCustomizedAlertsFeService, 'show');
     const deleteComplementary = jest.spyOn(component.api.resultsSE, 'DELETEcomplementaryinnovation').mockReturnValue(of({}));
     const emitSpy = jest.spyOn(component.saveedit, 'emit');
-    component.Ondelete(id);
-    expect(alertFeSpy).toHaveBeenCalled();
-    expect(deleteComplementary).toHaveBeenCalledWith(id);
-    expect(component.status).toBe(false);
-    expect(emitSpy).toHaveBeenCalledWith(true);
+    component.Ondelete(id, () => {
```

Comment:
@yecksin please take into consideration the way the methods are being named. `Ondelete` should not start with a capital letter. I recommend researching onto the linter rule [naming-convention](https://typescript-eslint.io/rules/naming-convention/)